matching/score.go

```diff
@@ -20,7 +21,17 @@ func Score(places []Place) float64 {
 	distances := calDistances(places)                     // Haversine distances
 	maxDist := math.Max(0.001, calMaxDistance(distances)) // protect against maximum distance being zero
 	avgDistance := stat.Mean(distances, nil) / maxDist    // normalized average distance
+	avgScore := avgPlacesScore(places)
+
+	return avgScore - avgDistance
```

Comment:
we are not worried about absolute values rite ? Negative values are also ok (as earlier)?

Reply:
we are okay with negative values. When the final score is negative, it just means the penalty from distance is greater than the positive gains of plan locations themselves. 
Sry about the late reply, I just saw your comments.